client/llb/exec.go

```diff
@@ -268,9 +289,15 @@ func AddMount(dest string, mountState State, opts ...MountOption) RunOption {
 	}
 }
 
+func ReadonlyRootFS(ei ExecInfo) ExecInfo {
```

Comment:
LGTM but please consider adding godoc about that OutputIndex skips ro mount?

Reply:
I agree that lot more docs are needed but this case it seems implementation detail. If ro outputs are not skipped it would still be a valid `llb`, just execute slightly slower.

Reply:
ok